IP_POLICY_GUIDANCE.md

```diff
@@ -0,0 +1,71 @@
+# OpenJS Foundation IP Policy Guidance
+
+The purpose of this guide is to make it straightforward for projects
+to follow the [OpenJS Foundation IP Policy][IP Policy].
+
+Of course, the source of truth remains the [IP Policy][].
+
+If in doubt about anything related to IP after reading this document,
+first read the [IP Policy][], and then [ask for help](#getting-help).
+
+## Basic requirements
+
+There are three key requirements for all OpenJS Foundation projects:
+
+1. [Licensing](#1-licensing),
+2. [copyright notices](#2-copyright-notices), and
+3. displaying the [standard website footer](#3-standard-website-footer) on the project's website.
+
+Additionally, projects may decide to adopt the [OpenJS Foundation Contributor License Agreement (CLA)](#contributor-license-agreement-cla).
```

Comment:
Should this not actually be a fourth basic requirement. `Incoming code must be made under a DCO or optionally an CLA` instead?

Reply:
See that this is what @kborchers pointed out.

Reply:
I've got a patch coming that does this, yes :-)

Reply:
Yup. This has been clarified.